test/kwokctl/metric.yaml

```diff
@@ -0,0 +1,49 @@
+kind: Metric
+apiVersion: kwok.x-k8s.io/v1alpha1
+metadata:
+  name: fake-node
+spec:
+  # I'm not sure if there is a way to configure the `metric-server` to change the path of the node metrics collection
+  path: "/metrics/nodes/{{ .metadata.name }}"
+  metrics:
+    - name: kubelet_node_name
+      help: "[ALPHA] The node's name. The count is always 1."
+      kind: gauge
+      labels:
+        - name: node
+          value: '{{ .metadata.name }}'
+      value: '1.0'
```

Comment:
Used 1.0 here, since the `cel` would evaluate it to an integer. Another approach I can take is to check the type and convert it. Not sure which one to take since the problem also exists on `value` field when you want to divide... Basically, you either specify `10.0` for example, or `double(10)`

Reply:
The evaluation values can be converted in this way

``` golang
	switch v := refVal.(type) {
	case types.Duration:
		return float64(v.Duration), nil
	case types.Int:
		return float64(v), nil
	case types.Double:
		return float64(v), nil
	case types.Uint:
		return float64(v), nil
	case types.Bool:
		if v {
			return 1, nil
		}
		return 0, nil
	default:
		return 0, fmt.Errorf("unsupported metric value type: %T", v)
	}
```